[NETBEANS-1074] : module review web.jsf

rtaneja1 opened the pull request:
Removed custom zip files under external, added resources from custom zips back under git. Added licenseinfo, corrected licenses in .ftl files.

matthiasblaesing commented:
Looks good in general - Three things I noticed:

### Bad/non-existing headers - no indication of the license
web-facesconfig_1_0.dtd: 
web-facesconfig_1_1.dtd:
Please download fresh copies from here: http://www.oracle.com/webfolder/technetwork/jsc/dtd/index.html
There are the correct headers added GPL+CDDL Version (mentioning that you did that in the commit message would be great, so that it is documented and noone can suspect tinkering)

### Missing info in the xml.xsd
I think it would be good to add the information of the source of the xml.xsd file. Please see `enterprise/web.core/src/org/netbeans/modules/web/taglib/resources/xml.xsd
xml.xsd` for a sample

### AnnotationsTest.java
Before the PR I had a test pass rate of 57passed, 1 failed, 2 errors. With your change I gets down to 55/3/2. When I revert your change, I'm back to 57/1/2. Could you please recheck that change?

rtaneja1 commented:
Thanks Matthias, compile issue in AnnotationTest.java goes away after module web.jsf20 is built. Now test results match 57/1/2.
Replaced web-facesconfig_1_0.dtd, web-facesconfig_1_1.dtd with ones from above link.
Updated xml.xsd with license header and source info from xml.xsd in web.core.
